src/Tribe/Main.php

```diff
@@ -189,57 +201,97 @@ public function maybe_set_common_lib_info() {
 	}
 
 	/**
-	 * Finalize the initialization of this plugin
+	 * Resets the global common info back to TEC's common path
+	 *
+	 * @since 4.10.6.2
 	 */
-	public function plugins_loaded() {
+	private function reset_common_lib_info_back_to_tec() {
+		// if we get in here, we need to reset the global common to TEC's version so that we don't cause a fatal
+		$tec         = Tribe__Events__Main::instance();
+		$main_source = file_get_contents( $tec->plugin_path . 'common/src/Tribe/Main.php' );
+
+		// if there isn't a VERSION, don't override the common path
+		if ( ! preg_match( $this->common_version_regex, $main_source, $matches ) ) {
+			return;
+		}
 
+		$GLOBALS['tribe-common-info'] = [
+			'dir'     => "{$tec->plugin_path}common/src/Tribe",
+			'version' => $matches[1],
+		];
+	}
+
+	/**
+	 * Prevents bootstrapping and autoloading if the version of TEC that is running is too old
+	 *
+	 * @since 4.10.6.2
+	 */
+	public function maybe_bail_if_old_tec_is_present() {
 		// early check for an older version of The Events Calendar to prevent fatal error
-		if (
-			class_exists( 'Tribe__Events__Main' ) &&
-			! version_compare( Tribe__Events__Main::VERSION, $this->min_tec_version, '>=' )
-		) {
-			add_action( 'admin_notices', [ $this, 'tec_compatibility_notice' ] );
-			add_action( 'network_admin_notices', [ $this, 'tec_compatibility_notice' ] );
-			add_action( 'tribe_plugins_loaded', [ $this, 'remove_exts' ], 0 );
-			/*
-			* After common was loaded by another source (e.g. The Event Calendar) let's append this plugin source files
-			* to the ones the Autoloader will search. Since we're appending them the ones registered by the plugin
-			* "owning" common will be searched first.
-			*/
-			add_action( 'tribe_common_loaded', [ $this, 'register_plugin_autoload_paths' ] );
+		if ( ! class_exists( 'Tribe__Events__Main' ) ) {
+			return;
+		}
 
+		if ( version_compare( Tribe__Events__Main::VERSION, $this->min_tec_version, '>=' ) ) {
 			return;
 		}
 
-		// WordPress and PHP Version Check
-		if ( ! self::supported_version( 'wordpress' ) || ! self::supported_version( 'php' ) ) {
-			add_action( 'admin_notices', array( $this, 'not_supported_error' ) );
-			add_action( 'network_admin_notices', array( $this, 'not_supported_error' ) );
+		$this->should_prevent_autoload_init = true;
+
+		add_action( 'admin_notices', [ $this, 'tec_compatibility_notice' ] );
+		add_action( 'network_admin_notices', [ $this, 'tec_compatibility_notice' ] );
+		add_action( 'tribe_plugins_loaded', [ $this, 'remove_exts' ], 0 );
+		/*
+		* After common was loaded by another source (e.g. Event Tickets) let's append this plugin source files
```

Comment:
Minor, but, should this be "_After common was loaded by another source (e.g. **The Events Calendar**)_ ?

Reply:
...not worth blocking the change, though.